### DISCUSSION TITLE
Github reset my password and I lose my account! More than 90 days and still no response!

### DISCUSSION THREAD

**Discussion by BinarywoodB** [community/community#50787] [Other Feature Feedback, Questions, & Ideas]
I have a GitHub account dilin-MS which is linked to my enterprise email.

After several months not logged in GitHub with this account, I suddenly receieved an email from GitHub Support <no-reply@github.com> on 12/7/2022 saying:
```
We’re writing to let you know that we observed suspicious activity that suggests a threat actor used a Personal Access Token (PAT) associated with your account to access private repository metadata.

Out of an abundance of caution, we reset your account password and revoked all of your Personal Access Tokens (classic), OAuth App tokens, and GitHub App tokens to protect your account, dilin-MS. 
```

The email also recommend me to review the security log. However, since my password and all PAT or mobile OAuth app is disabled by GitHub, I can neither login my account nor reset my credential. How am I supposed to get back my account???

I write several email to GitHub support but they create ticket and no response for more than 60 days! (Ticket ID: 1995588 / 1981887)

`dilin-MS` is my enterprise linked GitHub Account and is important for me. Please help me find my account back.

**Comment by BinarywoodB** [comment]
Hi @canuckjacq, could you help? This looks similar to this question: https://github.com/orgs/community/discussions/23989

My dilin-MS account is an enterprise account and I need to use it to access company repos. Not know how to get it back.

**Reply by BinarywoodB** [reply]
Anyone please help?

**Reply by ghost** [reply]
bro did you get your account back??

**Comment by yagodorea** [comment]
Seems you have a baby octopus, congrats!!! 9 months without answer is *CRAZY*
